resources/views/components/button.blade.php

```diff
@@ -1,6 +1,7 @@
-<button {{ $attributes->merge([
+@if(isset($href))
```

Comment:
Don't need to use an if to switch between `a` and `button` tags, just replace the HTML tag
`<{{ $tag }} ..... > .... </{{ $tag }}>`

So, we can use
`<x-button tag="a" href="google.com" .... />`

Reply:
Add a tag option